JapaneseFilter/sections/antiadblock.txt

```diff
@@ -8,6 +8,9 @@
 ! Bad: ||ad.doubleclick.net^ (should be in adservers.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/132696
+24u.jp#$#body { overflow: visible !important; }
+24u.jp#$#body div.fc-ab-root { display: none !important; }
```

Comment:
You can close this window, so what you need to do:
1. Remove your rules.
2. Open AdGuard Annoyances - antiadblock.txt
3. Find the same rules under this comment
```
! Better CSS. Scriptlet does not work on some sites (it's useful for uBO, if 'fundingchoicesmessages.google.com' is blocked).
```
4. Add domain `24u.jp` to them
5. Also add `@@||24u.jp^$generichide` for content blockers - you'll see it under the comment and the rule `! ##body div.fc-ab-root`.

Reply:
I see. Thank you for tell me guide.